src/app/business/block/file-share-detail/file-share-detail.component.ts

```diff
@@ -216,12 +216,16 @@ export class FileShareDetailComponent implements OnInit{
             str.forEach(item=>{
                 if(item.fileshareId == fileShareId){
                     item.createdAt = Utils.formatDate(item.createdAt);
+                    item.date = new Date(Utils.formatDate(item.createdAt)).getTime();
                     if(!item.description){
                         item.description = "--";
                     }
                     this.snapshots.push(item); 
                 }
             })
+            this.snapshots.sort((a,b)=>{
```

Comment:
please use variable names that are readable. Using `a` and `b` is not very helpful.

Reply:
I have just modified it. Please check it.